.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+__pycache__
```

Comment:
should contain more file patterns: .pyc etc
I recommend to use this file: https://github.com/github/gitignore/blob/main/Python.gitignore